quint/src/repl.ts

```diff
@@ -616,7 +616,10 @@ function tryEval(out: writer, state: ReplState, newInput: string): boolean {
       if (ex.kind === 'bool' && ex.value) {
         // A Boolean expression may be an action or a run.
         // Save the state, if there were any updates to variables.
-        const missing = state.evaluator.shiftAndCheck()
+        const [shifted, missing] = state.evaluator.shiftAndCheck()
+        if (shifted && verbosity.hasDiffs(state.verbosity)) {
+          console.log(state.evaluator.trace.renderDiff(terminalWidth(), { collapseThreshold: 2 }))
```

Comment:
We normally use `out` instead of `console.log` here, but `out` makes this be captured by the unit test output check, and that would add unnecessary complexity to the tests. I'd rather test this only with integration tests.

The bad part is that using `console.log` makes us print stuff in the middle of testing, so `npm test` gets a bit polluted.